addons/main.go

```diff
@@ -159,7 +163,7 @@ func main() {
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
 		Scheme:                 scheme,
 		MetricsBindAddress:     flags.metricsAddr,
-		Port:                   9453,
+		Port:                   9865,
```

Comment:
Supervisor cluster permits only certain ports(perhaps in 98xx range?). This one is available.

Reply:
**thought**: Probably later we should make the port configurable.